src/applications/vaos/tests/new-appointment/components/ReviewPage/index.cc-request.unit.spec.jsx

```diff
@@ -601,3 +614,172 @@ describe('VAOS <ReviewPage> CC request with provider selection', () => {
     expect(screen.history.push.called).to.be.false;
   });
 });
+
+describe('VAOS <ReviewPage> CC request with VAOS service', () => {
+  let store;
+
+  beforeEach(() => {
+    mockFetch();
+    store = createTestStore({
+      ...initialStateVAOSService,
+      user: {
+        profile: {
+          facilities: [{ facilityId: '983', isCerner: false }],
+          vapContactInfo: {
+            residentialAddress: {
+              addressLine1: '123 big sky st',
+              city: 'Cincinnati',
+              stateCode: 'OH',
+              zipCode: '45220',
+              latitude: 39.1,
+              longitude: -84.6,
+            },
+          },
+        },
+      },
+      newAppointment: {
+        pages: {},
+        data: {
+          facilityType: FACILITY_TYPES.COMMUNITY_CARE,
+          typeOfCareId: '323',
+          phoneNumber: '1234567890',
+          email: 'joeblow@gmail.com',
+          reasonAdditionalInfo: 'I need an appt',
+          communityCareSystemId: '983',
+          preferredLanguage: 'english',
+          hasCommunityCareProvider: true,
+          selectedDates: ['2020-05-25T00:00:00.000', '2020-05-26T12:00:00.000'],
+          communityCareProvider: {
+            resourceType: 'Location',
+            identifier: [
+              {
+                system: 'PPMS',
+                value: 'ppmsid',
+              },
+            ],
+            address: {
+              line: ['1012 14TH ST NW STE 700'],
+              city: 'WASHINGTON',
+              state: 'DC',
+              postalCode: '20005-3477',
+            },
+            name: 'CAMPBELL, WILLIAM',
+          },
+          bestTimeToCall: {
+            morning: true,
+            afternoon: true,
+            evening: true,
+          },
+        },
+        clinics: {},
+        ccEnabledSystems: [
+          {
+            id: '983',
+          },
+        ],
+        facilities: {},
+      },
+    });
+  });
+  afterEach(() => resetFetch());
+
+  it('should submit successfully', async () => {
+    mockAppointmentSubmit({
+      id: 'fake_id',
+    });
+    mockPreferences(null);
+
+    const screen = renderWithStoreAndRouter(<ReviewPage />, {
+      store,
+    });
+
+    await screen.findByText(/requesting a community care appointment/i);
+
+    userEvent.click(screen.getByText(/Request appointment/i));
+    await waitFor(() => {
+      expect(screen.history.push.lastCall.args[0]).to.equal(
+        '/requests/fake_id?confirmMsg=true',
+      );
+    });
+
+    const submitData = JSON.parse(global.fetch.getCall(0).args[1].body);
+
+    expect(submitData).to.deep.equal({
```

Comment:
I put this check in here rather than in the formSubmitTransformer unit tests, since I think this makes more sense and reduces duplicative test data.